macros/commands/get_tables_without_model.sql

```diff
@@ -0,0 +1,59 @@
+{% macro get_tables_without_models(deprecated_models_path=none) %}
```

Comment:
From this macro you could infer that it's somewhat related to sourced, snapshots, seeds, etc.
Maybe call it something in the area of `leftover` or `dangling`?
Not super critical I suppose.

Reply:
i like dangling, on it